Representor/HTTP/Adapters/HTTPSirenAdapter.swift

```diff
@@ -153,8 +142,17 @@ public func serializeSiren(representor:Representor<HTTPTransition>) -> [String:A
     representation["properties"] = representor.attributes
   }
 
-  if representor.transitions.count > 0 {
-    representation["actions"] = map(representor.transitions, transitionToSirenAction)
+  let links = filter(representor.transitions) { $1.method == "GET" }
```

Comment:
How about `link_transitions` just to keep in everything in the transitions. The other could be `action_transitions` if you wanted. Just a thought.